docs/source/docker/advanced/active_learning.rst

```diff
@@ -0,0 +1,222 @@
+Active learning
+==============================================
+
+For running an active learning step with the Lightly docker, we need to perform
+3 steps:
+
+1. Create an `embeddings.csv` file with the Lightly docker.
+2. Add your active learning scores as an additional column.
+3. Use the Lightly docker to perform a active learning sampling on the scores.
+
+
+
+Step 1: Create an embeddings.csv file with the Lightly docker.
+---------------
+Run the docker as usual and as described in the getting started section.
+The only difference is that you set the number of samples to be sampled to 1.0,
+this prevents sampling.
+E.g. create and run a bash script with the following content:
+
+.. code::
+
+    # Have this in a step_1_run_docker_create_embeddings.sh
+    INPUT_DIR=/path/to/your/dataset
+    SHARED_DIR=/path/to/shared
+    OUTPUT_DIR=/path/to/output
+
+    TOKEN= # put your token here
+    N_SAMPLES=1.0
+
+    docker run --gpus all --rm -it \
+      -v ${INPUT_DIR}:/home/input_dir:ro  \
+      -v ${SHARED_DIR}:/home/shared_dir:ro \
+      -v ${OUTPUT_DIR}:/home/output_dir \
+      lightly/sampling:latest \
+      token=${TOKEN} \
+      lightly.loader.num_workers=4     \
+      stopping_condition.n_samples=${N_SAMPLES}\
+      method=coreset \
+      remove_exact_duplicates=True     \
+      enable_corruptness_check=True \
+      enable_training=True     \
+      lightly.trainer.max_epochs=20 \
+      dump_dataset=False \
+      upload_dataset=False
+
+Running it will create a terminal output similar to the following:
+
+.. code-block::
+
+    [2021-09-28 15:47:34] Loading initial dataset...
+    [2021-09-28 15:47:34] Found 372 input images in input_dir.
+    [2021-09-28 15:47:34] Lightly On-Premise License is valid
+    [2021-09-28 15:47:34] Checking for corrupt images (disable with enable_corruptness_check=False).
+    Corrupt images found: 0: 100%|███████████████████████| 372/372 [00:01<00:00, 294.46it/s]
+    [2021-09-28 15:47:36] Embedding images.
```

Comment:
as a side note: your output doesn't show the training

Reply:
Yes, I set the number of epochs to 0.